webapp/app/js/controllers/main.js

```diff
@@ -68,6 +69,16 @@ treeherder.controller('MainCtrl', [
                         thEvents.selectPreviousUnclassifiedFailure
                     );
 
+                }else if(ev.keyCode == 39){
+                    //Highlight next next keys:<right arrow>
+                    $rootScope.$emit(
+                        thEvents.selectNextJob
+                    );                    
```

Comment:
This line 76 and line 81 below, also have ~20 trailing whitespace characters that need to be removed.

Also for both, the entire emit can live on one line rather than three, as long as it doesn't exceed 80 characters in total length.

It should have a single character whitespace after and before the curly braces (see line 56 above as an example). All conditionals (if, if-else, while, etc) touched in this PR below should observe that same style, per the Mozilla style guides mentioned in [comment5](https://bugzilla.mozilla.org/show_bug.cgi?id=1089737#c5) in the bug.